prow/cluster/starter/plank.deployment.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: plank
+  labels:
+    app: plank
+spec:
+  replicas: 1 # Do not scale up.
```

Comment:
More than one instance during an upgrade is not guaranteed with Rolling deployments (Rolling is defaulted if you omit the strategy field). We are using a Recreate strategy, but it's only the upgrade part that is guaranteed to not run both a new and an old pod at the same time and you can still scale those up.

Reply:
IIRC this is verbatim from `prow/cluster/starter.yaml`.

Reply:
This is verbatim from the existing yaml which probably could also be corrected if inaccurate.

Reply:
I'm not grokking the feedback to fully understand what the desired change is?  I'm happy to make whatever edits both places.

Reply:
I think this is fine as-is but perhaps in a follow-up we should clarify it in all places, for now this matches the source material..